lib/benchee/formatters/markdown/helpers.ex

```diff
@@ -41,10 +41,14 @@ defmodule Benchee.Formatter.Markdown.Helpers do
     Enum.filter(suite.scenarios, fn scenario -> scenario.input_name == input end)
   end
 
+  def inputs(%{configuration: %{input_names: names}}), do: names
+
   def inputs(suite) do
     suite.configuration.inputs |> Enum.map(fn {name, _} -> name end)
```

Comment:
Just to be clear, I don't think these changes are needed - right? I mean, it's good to have them and change them but the way I kept `inputs` half-way in tact, should work with this - right? :eyes:

Reply:
Yes, that change is just for good to have and not really needed. I should add a comment here for my future self.

Reply:
Cool cool, just wanted to make sure my understanding of "technically this is a breaking change but practically it shouldn't be for most/all use cases" is right :)